### DISCUSSION TITLE
Delete Files After Deleting Record

### DISCUSSION THREAD

**Discussion by discoverlance-com** [filamentphp/filament#337] [Ideas]
I realized this, after deleting records of my posts, the images attached with the **RichEditor** component and even the ones created from the **FileUpload** component were not deleted and they were still existing on my drive, is there something I am missing. If not could this feature be added please, because I have to find a workaround or visit my storage system to manually delete uploaded files. If anyone knows a workaround too can you please let me know, Thanks.

**Comment by zltn** [comment]
See discussion here: https://github.com/laravel-filament/filament/issues/279

**Reply by discoverlance-com** [reply]
Thanks
